frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/details-card.tsx

```diff
@@ -34,8 +33,6 @@ const infrastructureResource: FirehoseResource = {
 
 const cephClusterResource: FirehoseResource = {
   kind: referenceForModel(CephClusterModel),
-  namespaced: true,
```

Comment:
@bipuladh we need to set the namespace to false if we want to skip the namespace.

Reply:
Noted.